src/app/components/BibPage/BibPage.jsx

```diff
@@ -40,7 +40,7 @@ const BibPage = (props) => {
   const marcRecord = isNYPLReCAP ? <MarcRecord bNumber={bNumber[0]} /> : null;
 
   return (
-    <DocumentTitle title={`${title} | Research Catalog`}>
+    <DocumentTitle title={`${title} | Shared Collection Catalog | NYPL`}>
```

Comment:
haha OK so if we already have an example for this. I am happy to update the bib with item's title.